miden-lib/asm/miden/kernels/tx/prologue.masm

```diff
@@ -384,6 +384,46 @@ proc.validate_new_account
     # => []
 end
 
+proc.validate_account_procedures
```

Comment:
@bobbinth Is this the right place to put this type of procedure, should I move it somewhere else in this file, or in another file?

Reply:
Yes - that's where I'd probably put it as well.